barretenberg/cpp/src/barretenberg/protogalaxy/combiner.test.cpp

```diff
@@ -19,7 +18,6 @@ TEST(Protogalaxy, CombinerOn2Instances)
     constexpr size_t NUM_INSTANCES = 2;
     using ProverInstance = ProverInstance_<Flavor>;
     using ProverInstances = ProverInstances_<Flavor, NUM_INSTANCES>;
-    using ProtoGalaxyProver = ProtoGalaxyProver_<ProverInstances>;
     using Fun = ProtogalaxyProverInternal<ProverInstances>;
```

Comment:
I dont quite like the Fun pattern (unless it's a std pattern and I don't know of it?) mb PGInternal or ProverInternal might be better. My thinking is if I see `Fun::*` I'll always have to go to the using declaration to figure out what class is used.

Reply:
Two options are:
 1) just make function templates, not static functions in a class template, or  
 2) just put these functions in the prover.

The second has the disadvantage that we'd have to declare these in the header to use them in the imple, and I think it's nicer to have the header contain fewer declarations to be more clear about what the class does. Will go with the first approach in a follow-on